src/navigators/Main.tsx

```
@@ -39,14 +39,19 @@ import ServiceManager from '@services/ServiceManager';
 import { UpdateContextProvider } from '@components/Context/UpdateContext';
+import { MMKVStorage } from '@utils/mmkv/mmkv';
 const Stack = createNativeStackNavigator<RootStackParamList>();
 
 const MainNavigator = () => {
   const theme = useTheme();
   const { updateLibraryOnLaunch } = useAppSettings();
   const { refreshPlugins } = usePlugins();
-  const [isOnboarded] = useMMKVBoolean('IS_ONBOARDED');
-
+  const [isOnboarded] = useMMKVBoolean('IS_ONBOARDED', MMKVStorage);
+  console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)